test/framework/src/main/java/org/opensearch/test/OpenSearchTestCase.java

```diff
@@ -225,6 +227,7 @@ public static void resetPortCounter() {
 
     @Override
     public void tearDown() throws Exception {
+        Schedulers.shutdownNow();
```

Comment:
I know nothing about `Schedulers` here, but it looks like an executor. Does the security manager play well with this?